tests/admin-mode_test.go

```diff
@@ -0,0 +1,273 @@
+			//Also Printing the logs of the experiment
+			By("Waiting for job completion")
+			jobPodLogs, err := utils.JobLogs(experimentName, chaosTypes.ChaosNamespace, engineName, client)
+			Expect(jobPodLogs).To(Equal(0), "Failed to print the logs of the experiment")
+			Expect(err).To(BeNil(), "Failed to get the experiment job pod")
+
+			//Checking the chaosresult
+			By("Checking the chaosresult")
+			chaosResult, err := clientSet.ChaosResults(chaosTypes.ChaosNamespace).Get(engineName+"-"+experimentName, metav1.GetOptions{})
+			Expect(string(chaosResult.Status.ExperimentStatus.Verdict)).To(Equal("Pass"), "Verdict is not pass chaosresult")
+			Expect(err).To(BeNil(), "Failed to get chaosresult")
+		})
+	})
+	// BDD TEST CASE 2
+	// Operator in litmus ns, engine and experiment is in test ns
+	// Application is in deafult ns
+	Context("Check for litmus components", func() {
+
+		It("Should check for creation of runner pod", func() {
+
+			//Create Namespace for the test
+			By("Creating namespace for the test")
+			cmd := exec.Command("kubectl", "create", "ns", "test")
+			cmd.Stderr = os.Stderr
+			err := cmd.Run()
+			if err != nil {
+				log.Fatalf("cmd.Run() failed with %s\n", err)
+			}
+
+			//Installing RBAC for experiment that is pod-delete
+			rbacPath := "https://raw.githubusercontent.com/litmuschaos/pages/master/docs/litmus-admin-rbac.yaml"
+			err = exec.Command("wget", "-O", experimentName+"-sa.yaml", rbacPath).Run()
+			Expect(err).To(BeNil(), "Fail to fetch rbac")
+			err = exec.Command("sed", "-i",
+				`s/namespace: litmus/namespace: test/g;
+			     s/name: litmus-admin/name: admin-rbac/g`,
+				experimentName+"-sa.yaml").Run()
+			err = exec.Command("kubectl", "apply", "-f", experimentName+"-sa.yaml").Run()
+			Expect(err).To(BeNil(), "Fail to create RBAC")
+			fmt.Println("Rbac has been created successfully !!!")
+
+			//Creating Chaos-Experiment for pod delete
+			By("Creating Experiment for pod delete")
+			err = exec.Command("wget", "-O", "pod-delete.yaml", "https://hub.litmuschaos.io/api/chaos?file=charts/generic/pod-delete/experiment.yaml").Run()
+			Expect(err).To(BeNil(), "fail to get chaos experiment")
+			err = exec.Command("sed", "-i", `s/litmuschaos\/ansible-runner:latest/`+chaosTypes.ExperimentRepoName+`\/`+chaosTypes.ExperimentImage+`:`+chaosTypes.ExperimentImageTag+`/g`, "pod-delete.yaml").Run()
+			Expect(err).To(BeNil(), "Failed to edit chaos experiment yaml")
+			err = exec.Command("kubectl", "apply", "-f", "pod-delete.yaml", "-n", "test").Run()
+			Expect(err).To(BeNil(), "Failed to create chaos experiment")
+			fmt.Println("Pod delete Chaos Experiment Created Successfully")
+
+			//Installing chaos engine for the experiment
+			//Fetching engine file
+			By("Fetching engine file for pod delete experiment")
+			err = exec.Command("wget", "-O", experimentName+"-ce.yaml", "https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/generic/pod-delete/engine.yaml").Run()
```

Comment:
We already get this file in BDD1, so it is already there in local

Reply:
same as above